libethereum/Account.h

```diff
@@ -228,7 +228,6 @@ class AccountMask
 
 	bool allSet() const { return m_hasBalance && m_hasNonce && m_hasCode && m_hasStorage; }
 
-private:
```

Comment:
instead of removing `private`, define getter/setter in case you need direct access